scripts/validate_my_definition.py

```diff
@@ -26,6 +26,8 @@ def validate_definition(definition_file):
     except AlertLogicOpenApiValidationException as e:
         print(f"Validation has failed - definition has failed AlertLogic specific check {e}")
         exit(1)
+    except AlertLogicOpenApiValidationWarning as e:
+        print(f"***WARNING***: definition may not meet criteria {e}")
```

Comment:
Suggest the simpler / more straightforward

`***WARNING***: {e}` or 

`***WARNING***: Validation warning {e}`